IronFrame.Test/IronFrame.Test.csproj

```diff
@@ -33,6 +33,10 @@
     <WarningLevel>4</WarningLevel>
   </PropertyGroup>
   <ItemGroup>
+    <Reference Include="Interop.NetFwTypeLib">
```

Comment:
I'm guessing that it's complaining because you have a hard-coded path to the `Debug` output folder and MSBuild builds `Release` builds.

I would move `Interop.NetFwTypeLib.dll` to a `vendor` or `lib` folder at the root.  I'm pretty sure we already have this somewhere that works because the non-test code was already referencing the DLL and building fine under MSBuild.